Dependencies/xr/CMakeLists.txt

```diff
@@ -46,6 +46,10 @@ else()
     target_link_libraries(xr PRIVATE openxr_loader)
 endif()
 
+target_link_libraries(xr
+    PUBLIC arcana
+    PUBLIC JsRuntime)
```

Comment:
Worked with Ryan offline, and got rid of the need to use another scheduler.  There was a bug in our use of ticketed collection using a mutex instead of a recursive_mutex that was causing my other issues, and we moved the initialization code to the first instance of GetNextFrame to get around the OpenGL issues.